### DISCUSSION TITLE
UI design / Deciding small things/ A ton of numbers :)

### DISCUSSION THREAD

**Discussion by SearWaizy** [InaSLew/AutumnGameJam-PumpkinMadness#21] [General]
**Player health / movement / invulnerability state**
* Choose your own player speed for moving right and left, jump speed and jump hight, should be "medium-fast" (just a placeholder).
* The player has 3 "hearts", the player dies on the third hit (just a placeholder).
* After the player has recived damage the invulnerability state will last for 2 sec (just a placeholder).

**Player sword**
* Range; 1 unit at the dicrection the player is facing. 
* One stab is instant, the cooldown before the player can stab again is 0.5 sec so the player dosent spam the attacks (just a placeholder).

**Enemys health and movement**
* Both "Flying enemy and Ground enemy" will die after **one stab** from the player's sword.
* Choose your own speed for moving right and left, jump speed and jump hight, should be "medium-slow" (just a placeholder).

**Enemys spawn rate**
* Enemys spawn rate is in 0.7 seconds intervals. 
* Tutorial sequence is 5 enemys (3 "Ground enemys" and "2 flying enemys") , the game will only progress until all of the 5 enemys has been killed by the player.
* After tutorial sequence is finished the enemys start spawning with a 20% cap increase each 20 seconds. 

Ex; 10 enemys spawns with a enemy cap of 10, after 20 seconds the enemy cap will increase with 20%, therefore increasing the enemy cap to 12 (10 * 120% = 12). After 20 seconds the cap will increase once more with 20%, increasing the enemy cap to 14 (12 * 120% = 14.4) this will continue  until the player dies.

* Max cap 100 enemys on the screen. (Just a placeholder).

**UI**
* Posted on Discord.

**Comment by InaSLew** [comment]
Very good dev references! thx ✨
